vendor/github.com/submariner-io/cloud-prepare/pkg/azure/ocpgwdeployer.go

```diff
@@ -39,7 +39,7 @@ import (
 )
 
 const (
-	submarinerGatewayGW      = "subgw"
+	submarinerGatewayGW      = "-subgw-"
```

Comment:
Agree that this is part of the vendor folder, but it can create multiple GWs during upgrades. Do we handle that @aswinsuryan ?

Reply:
There is a check to if the active gateways match the required number of gateways, If yes cloud-prepare won't create a new g/w.